plugins/pushes/atlas/push.rb

```diff
@@ -27,8 +27,10 @@ def execute(uploader)
         cmd << "-vcs" if config.vcs
         cmd += config.includes.map { |v| ["-include", v] }
         cmd += config.excludes.map { |v| ["-exclude", v] }
+        cmd += metadata.map { |k,v| ["-metadata", "#{k}=#{v}"] }
         cmd += ["-address", config.address] if config.address
         cmd += ["-token", config.token] if config.token
+        cmd += ["-message", message] if message
```

Comment:
Re: my previous comment, I need to get the value for that option in here somehow...